forms/jqm/element.js

```diff
@@ -68,9 +71,13 @@ define(function (require) {
     renderLabel: function () {
       if (!this.$label) {
         this.$label = $('<label class="ui-input-text"></label>');
+      }
+
+      if (!this.$el.find(this.$label).length) {
         this.$el.append(this.$label);
       }
-      this.$label.html(this.model.attributes.label || '');
+
+      this.$label.text(this.model.attributes.label || '');
```

Comment:
special characters will be printed as is. The platform sends any special char like ampersands down unencoded, but i can change it back to .html to be sure any special chars are translated correctly